components/brave_wallet_ui/components/desktop/wallet-page-layout/style.ts

```diff
@@ -5,15 +5,15 @@ export const StyledWrapper = styled('div') <{}>`
   align-items: center;
   justify-content: center;
   background-color: ${(p) => p.theme.color.background02};
-  height: 100vh;
+  height: 100%;
```

Comment:
Instead of changing height to `100%` can we add 
`overflow-y: scroll;`
`overflow-x: hidden;`
to prevent clipping at the bottom.

Reply:
I set `overflow-y` to `auto` to prevent displaying the scroll bar when there's no overflow.

Reply:
Sounds great! :) could you also just change the `height` back to `100vh` in the `StyledWrapper`

Reply:
Fixed!